.github/workflows/sonar-cloud-pipeline.yml

```diff
@@ -48,5 +48,10 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         run: |
+          sudo yum -y install java-17-openjdk.x86_64
+          sudo yum -y install java-17-openjdk-devel.x86_64
+          sudo alternatives --set java java-17-openjdk.x86_64
+          sudo alternatives --set javac java-17-openjdk.x86_64
+          sudo alternatives --set jre_openjdk java-17-openjdk.x86_64
           chown -R aswfuser:aswfgroup .
-          su -c "cd cuebot && ./gradlew jacocoTestReport sonarqube -Dsonar.login=$(SONAR_TOKEN)" aswfuser
+          su -c "cd cuebot && ./gradlew jacocoTestReport sonar aswfuser
```

Comment:
I think there's a missing quote here. Also, we don't need to define the sonar token?

Reply:
Oops. Fixed the missing quote. 

According to the documentation, having SONAR_TOKEN defined as an environment variable is supposed to be enough. Let's see when it makes it to master where we can actually test.